src/yival/enhancers/utils.py

```diff
@@ -28,22 +28,33 @@ def scratch_variations_from_str(
     target_str:
         This is the generated new output
         var1: hello world
+        hello world
         var2: bye
     
     variations: ["var1", "var2"]
 
     function result:
     {
-        "var1": "hello world"
+        "var1": "hello world\nhello world"
         "var2": "bye"
     }
     """
     result = {}
```

Comment:
ensure that all the variations are present in the dictionary right from the start, even if they are not found in 'target_str'
result = {var: "" for var in variations}

Reply:
yeah , you're right 😺